bridge-cli/src/fast_bridge_command.rs

```diff
@@ -0,0 +1,151 @@
+use crate::{combined_config, CliConfig, Network};
+use clap::Subcommand;
+use ethers_core::types::{Address, TxHash};
+use fast_bridge::{FastBridge, FastBridgeBuilder};
+use near_primitives::types::AccountId;
+use std::str::FromStr;
+
+#[derive(Subcommand, Debug)]
+pub enum FastBridgeSubCommand {
+    Transfer {
+        #[clap(short, long)]
+        token: String,
+        #[clap(short, long)]
+        amount: u128,
+        #[clap(short, long)]
+        fee: u128,
+        #[clap(short, long)]
+        eth_token_address: String,
+        #[clap(short, long)]
+        recipient: String,
+        #[clap(short, long)]
+        valid_till: u64,
```

Comment:
Calculating this parameter manually is inconvenient. It is better to calculate it automatically and specify the time here relative to the current moment, preferably in seconds/minutes.

Reply:
What's the default fast bridge timeout?

Reply:
It is now optional and is set to 30 minutes by default 33bd403ff4180b98574f99ca6be01f3473475daf